lib/tailwindcss/engine.rb

```diff
@@ -2,6 +2,9 @@
 
 module Tailwindcss
   class Engine < ::Rails::Engine
+    config.tailwindcss = ActiveSupport::OrderedOptions.new
+    config.tailwindcss.server_process = false # Rails.env.development?
```

Comment:
If (or when) you decide to make "server process" the default for all:

- Just set the `tailwindcss.server_process` option value to `Rails.env.development?` ([propshaft does it like that](https://github.com/rails/propshaft/blob/d75d9d8547bc8eddd85045311fc426b8d7e74897/lib/propshaft/railtie.rb#L19))
- Optionally remove the `if DEVELOPMENT_ENVIRONMENT_CONFIG_PATH.exist?` section from `lib/install/tailwindcss.rb`